gem/lib/pupperware/spec_helper.rb

```diff
@@ -123,9 +123,16 @@ def docker_compose(command_and_args, stream: StringIO.new)
     overrides = IS_WINDOWS ?
                   'docker-compose.windows.yml' :
                   'docker-compose.override.yml'
+    services_path = Pathname.new(File.join(__dir__, 'compose-services'))
+    additional_services = (ENV['ADDITIONAL_COMPOSE_SERVICES'] || '').split(',').map do |service_name|
```

Comment:
Using an ENV var here is not the greatest, but it's a low friction solution.

I don't expect too many more changes to the testing gym.

Reply:
Decided to do this in code instead - makes more sense to me than an ENV var.